### PR TITLE
📦 applying package updates ***NO_CI***

### PR DESCRIPTION
**Pull request by Saadnajmi**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Looked at the output of this command

> yarn beachball publish --no-publish --bump-deps -m"📦 applying package updates ***NO_CI***"  --verbose -b origin/manual-publish -n *** --access public -y

.. and then undid a bunch of it to manually only bump the packages I wanted bumped (avatar, badge, dependency-profiles. tester, and tester-win32)

### Verification

Went through updated package.json version numbers and compared with NPM

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/components/Button/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/button",
-  "version": "0.32.3",
+  "version": "0.32.4",
```
Comment (Saadnajmi) [inline packages/components/Button/package.json:3]: NPM says the last published is 0.32.3
https://www.npmjs.com/package/@fluentui-react-native/button

File: packages/components/Menu/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/menu",
-  "version": "1.0.82",
+  "version": "1.0.83",
```
Comment (Saadnajmi) [inline packages/components/Menu/package.json:3]: NPM says last published is 1.0.82

File: packages/components/MenuButton/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/menu-button",
-  "version": "0.9.7",
+  "version": "0.9.8",
```
Comment (Saadnajmi) [inline packages/components/MenuButton/package.json:3]: NPMJS says last published is 0.9.7

File: packages/experimental/Dropdown/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/dropdown",
-  "version": "0.6.0",
+  "version": "0.6.1",
```
Comment (Saadnajmi) [inline packages/experimental/Dropdown/package.json:3]: NPM says last published is 0.6.0

https://www.npmjs.com/package/@fluentui-react-native/dropdown

File: packages/experimental/ExperimentalIcon/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/experimental-icon",
-  "version": "0.0.4",
+  "version": "0.0.5",
```
Comment (Saadnajmi) [inline packages/experimental/ExperimentalIcon/package.json:3]: NPM says last published is 0.0.4

https://www.npmjs.com/package/@fluentui-react-native/experimental-icon

File: packages/experimental/MenuButton/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/experimental-menu-button",
-  "version": "0.5.24",
+  "version": "0.5.25",
```
Comment (Saadnajmi) [inline packages/experimental/MenuButton/package.json:3]: NPM says last published is 0.5.24

https://www.npmjs.com/package/@fluentui-react-native/experimental-menu-button

File: packages/libraries/core/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-native",
-  "version": "0.33.7",
+  "version": "0.33.8",
```
Comment (Saadnajmi) [inline packages/libraries/core/package.json:3]: NPM says last published is 0.33.7

https://www.npmjs.com/package/@fluentui/react-native

**Comment by rurikoaraki** [comment]
I think we should just manually bump the ones that have been published already, which is avatar, badge, dependency-profiles. tester, and tester-win32. Then we can fix the next round of publishing and everything will be aligned.

I worry about skipping over because then it's likely that the version numbers will stay out of sync.

**Comment by Saadnajmi** [comment]
> I think we should just manually bump the ones that have been published already, which is avatar, badge, dependency-profiles. tester, and tester-win32. Then we can fix the next round of publishing and everything will be aligned.
> 
> I worry about skipping over because then it's likely that the version numbers will stay out of sync.

Sounds good. I'll revert the extra bumps

**Comment by rurikoaraki** [comment]
I'm confused, I thought we were updating these packages: avatar, badge, dependency-profiles. tester, and tester-win32?

**Comment by Saadnajmi** [comment]
> I'm confused, I thought we were updating these packages: avatar, badge, dependency-profiles. tester, and tester-win32?

Yeah I think I did the reverse :( I'll have to redo this sorry. I'll convert to draft and mark it as ready when it's the right set.

**Comment by Saadnajmi** [comment]
@r

> > I'm confused, I thought we were updating these packages: avatar, badge, dependency-profiles. tester, and tester-win32?
> 
> Yeah I think I did the reverse :( I'll have to redo this sorry. I'll convert to draft and mark it as ready when it's the right set.

@rurikoaraki OK Through the magic of `git reset HEAD~` and some other voodoo, I think I've got it to the minimum necessary changes.

**Comment by Saadnajmi** [comment]
Judging from the CI, it looks like I also need to revert dependency version number bumps.

File: change/@fluentui-react-native-button-6e9b526b-44b9-44c1-a7d4-714054ace485.json [hunk 8]
```diff
@@ -1,7 +0,0 @@
-{
-  "type": "patch",
```
Comment (rurikoaraki) [inline change/@fluentui-react-native-button-6e9b526b-44b9-44c1-a7d4-714054ace485.json:2]: I think this should stay, this package hasn't been updated as part of the bump

File: change/@fluentui-react-native-experimental-icon-9d1fb407-06f2-4609-86c7-1b11f607c3ab.json [hunk 9]
```diff
@@ -1,7 +0,0 @@
-{
-  "type": "patch",
-  "comment": "Fixed import for the Icon",
-  "packageName": "@fluentui-react-native/experimental-icon",
```
Comment (rurikoaraki) [inline change/@fluentui-react-native-experimental-icon-9d1fb407-06f2-4609-86c7-1b11f607c3ab.json:4]: This one, too

File: change/@fluentui-react-native-button-6e9b526b-44b9-44c1-a7d4-714054ace485.json (hunk 8 above)
Comment (Saadnajmi) [reply change/@fluentui-react-native-button-6e9b526b-44b9-44c1-a7d4-714054ace485.json:2]: Ah, I think I'm just gonna redo this again as a whole and not depend on beachball and just bump numbers / delete change files manually..

Comment (Saadnajmi) [reply change/@fluentui-react-native-button-6e9b526b-44b9-44c1-a7d4-714054ace485.json]: Brought back

**Review by rurikoaraki — APPROVED** [review]

**Pull request merged**